conneg-by-ap/index.html

```diff
@@ -67,12 +67,24 @@ <h2>Introduction</h2>
       Arguments (<abbr title="Query String Arguments">QSAs</abbr>). QSAs are
       useful for humans and machines for situations where negotiation via HTTP is not practical, such
       as when manually entering requests into web browsers. This specification provides an abstract model
-      of content negotiation by profile and presents guidance on the use of two specific methods, HTTP &amp; QSA, that both conform to the abstract model meaning they are.
+      of content negotiation by profile and presents guidance on the use of two specific methods, HTTP &amp; QSA, that both conform to the abstract model.
     </p>
     <p>
-      While this specification only describes HTTP and QSA methods for content negotiation by profile, it is expected
-      that implementers will want to implement functionally equivalent methods for other environments, some of which may
-      not yet be realized; future environments. Implementers may add additional functions/features to their implementations for other environments as long as they ensure their implementation conforms to this specification's abstract model, as the HTTP and QSA methods do.
+      The HTTP and QSA methods are formally defined in a series of <em>Functional Profiles</em> of the Abstract Model.
+      While this specification only describes these Functional Profiles, it is expected
+      that implementers will want to implement methods for other environments, some of which may
+      not yet be realized; future environments.
+      To do so, implementers are encouraged to define additional Functional Profiles.
+      Implementers may add additional features to their implementations which they may define in their Functional Profiles
+      as long as they ensure they always conform to the Abstract Model.
+    </p>
+    <p>
+      In addition to taking as implementation examples the HTTP &amp; QSA Functional Profiles that this specification
+      defines, implementers are guided by the Alternate Representations Data Model which has concrete examples for
+      implementation in this specification's appendixes. Implementers will need to ensure that any Functional Profile
+      they define can represent alternate representations information as per this model. Such demonstration would be
+      best implemented by a published mapping between their environment's native alternate profiles model representation
+      and a concrete implementation of the model given here, such as the <a href="#altr-owl">OWL ontology representation</a>.
     </p>
     <p>
       While this specification only describes HTTP and QSA methods for content negotiation by profile, it is expected
```

Comment:
This text can be removed since it's already in lines 75-76

Reply:
I've re-written both overlapping paragraphs to improve wording and to de-duplicate.